cmake/external/abseil-cpp.cmake

```diff
@@ -38,3 +38,9 @@ if (GDK_PLATFORM)
   # tell Abseil to pretend we're building an APP.
   target_compile_definitions(absl_symbolize PRIVATE WINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP)
 endif()
+
+if(NOT onnxruntime_DISABLE_ABSEIL)
```

Comment:
nit: this doesn't need to be within `if (NOT onnxruntime_DISABLE_ABSEIL)` as this file is included only `if (NOT onnxruntime_DISABLE_ABSEIL)`